src/ContentBundle/Resources/views/Block/Layout/placeholder.html.twig

```diff
@@ -1,3 +1,9 @@
-<div class="pm-placeholder" {% if content|length <= 0 %}data-pm-empty="true"{% endif %} {{ manage_tags() }}>
+{% if tag|length > 0 %}
+    <{{ tag }} {% for key,value in data %}{{ key }}="{{ value }}"{% endfor %}{% if content|length <= 0 %}data-pm-empty="true"{% endif %} {{ manage_tags() }}>
+{% endif %}
```

Comment:
Could you remove the `if` statements here, and pass a default value (`div`) for `tag`?